src/modules/songs/forms/AddSongToSet/components/SetSelectionAllUpcomingSets.tsx

```
@@ -0,0 +1,144 @@
+        <React.Fragment key={pageIndex}>
+          {(!setsPage.sets || setsPage.sets.length === 0) && (
+            <SetSelectionSetItem
+              title="No sets found..."
+              subtitle="But you can create one!"
+            />
+          )}
+          {setsPage.sets.map((set) => (
+            <SetSelectionSetItem
+              key={set.id}
+              title={formatFriendlyDate(set.date)}
+              subtitle={set.eventType!} // eventType is marked as notNull:true, so not sure why the type is nullable, but this appeases TS for now
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)